packages/Menu/src/Menu.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { useState, useEffect, EventHandler } from 'react';
+import PropTypes from 'prop-types';
+import Popover, { Align, Justify } from '@leafygreen-ui/popover';
+import { emotion } from '@leafygreen-ui/lib';
+import { colors } from '@leafygreen-ui/theme';
+import { cx } from 'emotion';
+
+const { css } = emotion;
+
+const rootMenuStyle = css`
+  width: 202px;
+  border-radius: 3px;
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.15);
+  background-color: ${colors.mongodb.white};
+`;
+
+interface Props {
+  /**
+   * Determines the active state of the Menu.
+   *
+   * default: `false`
+   */
+  active: boolean;
+
+  /**
+   * Determines the alignment of the Menu relative to a trigger element.
+   *
+   * default: `bottom`
+   */
+  align: Align;
+
+  /**
+   * Determines the justification of the Menu relative to a trigger element.
+   *
+   * default: `end`
+   */
+  justify: Justify;
+
+  /**
+   * Content that will appear inside of the Menu.
+   */
+  children?: React.ReactElement;
+
+  /**
+   * className applied to Menu.
+   */
+  className?: string;
+
+  /**
+   * A reference to the element against which the Menu will be positioned.
+   */
+  refEl?: React.RefObject<HTMLElement>;
+
+  /**
+   * A slot for the element used to trigger the Menu. Passing a trigger allows
+   * Menu to control opening and closing itself internally.
+   */
+  trigger?: React.ReactElement;
+
+  /**
+   * Specifies that the popover content will appear portaled to the end of the DOM,
+   * rather than in the React subset of the DOM tree.
+   *
+   * default: `true`
+   */
+  usePortal?: boolean;
+}
+
+/**
+ * # Menu
+ *
+ * Menu component
+ *
+ * ```
+<button>
+  <Menu active={true}>
+    <MenuItem>Hello World!</MenuItem>
+  </Menu>
+</button>
+ * ```
+ * @param props.children Content to appear inside of Menu.
+ * @param props.active Boolean to describe whether or not Menu is active.
+ * @param props.className Classname applied to Menu.
+ * @param props.align Alignment of Menu relative to another element: `top`, `bottom`, `left`, `right`.
+ * @param props.justify Justification of Menu relative to another element: `start`, `middle`, `end`.
+ * @param props.refEl Reference element that Menu should be positioned against.
+ * @param props.usePortal Boolean to describe if content should be portaled to end of DOM, or appear in DOM tree.
+ * @param props.trigger Trigger element to set active state of Menu, makes component controlled
+ *
+ */
+
+function Menu({
+  align = 'bottom',
```

Comment:
Spread the {...rest} props onto Popover rather than the div below? Not really sure what you mean here

Reply:
We should be deliberate about which popover props Menu supports. We explicitly don’t want to support passing through the spacing prop, for example. 

I believe the only props we’re consistently going to want in components built with Popover are align, justify, and withoutPortal.

Reply:
^^ Still a little confused by what's meant here. Using pick/exclude (or, eventually, omit) to be deliberate about the exact props that MenuProps extends from PopoverProps, but not sure how that manifests over here.

Reply:
If we’re being selective about the popover props passed through, I’m not sure having a helper that pulls those props out of `rest` is better than just destructuring those props, since it may not be reusable.

Reply:
I think I'm leaning towards your side here @DesignerDave and disagreeing with myself. If we want to create a utility to make consistent the props we pass to Popover then we can in the future, but let's first see what becomes common before we commit to a generic helper for that.

Reply:
Ok so leave as is for now? @hswolff @DesignerDave

Reply:
Leave as is, yes please.

Reply:
👍